### PR TITLE
Tooltip: Rendering hidden hoverable area to allow for interaction with Tooltips

### PR DESCRIPTION
**Pull request by khmakoto**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

This PR adds the ability for `Tooltips` to render a hidden hoverable area around the `Tooltip` to allow for interaction with them and not be instantly dismissed if the user is not hovering specifically over the beak.

Below are some examples of where this area is being rendered depending on the `Tooltip` rendering direction.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9775)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
I talked to Mak about not relying on DirectionalHint in that there is no guarantee that the end result will be that direction. With only 1 tooltip on the page at a time, we're probably fine simply extending the target area on all 4 sides. If we can optimize this in the future that'd be great.

File: packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ export class TooltipBase extends React.Component<ITooltipProps, any> {
       theme: theme!,
       className: className || (calloutProps && calloutProps.className),
       delay: delay!,
+      gapSpace: (calloutProps && calloutProps.gapSpace && -15 - calloutProps.gapSpace) || -15,
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx]: we've already got defaultProps set, so you shouldn't need to check this, right? Does TS complain? Also, how about doing the -15 calc in the styles file.

Comment (khmakoto) [reply packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx]: TS was complaining, I defaulted to using `gapSpace?` and setting a the default to 0 in the `styles`. I just pushed that change which I think makes the code a little bit cleaner.

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx]: ah yeah, i was wondering if typescript would complain. Could always change it to have a default value in the destructuring, but no worries.

File: packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.styles.ts [hunk 2]
```diff
@@ -2,9 +2,11 @@ import { ITooltipStyleProps, ITooltipStyles, TooltipDelay } from './Tooltip.type
 import { AnimationClassNames } from '../../Styling';
 
 export const getStyles = (props: ITooltipStyleProps): ITooltipStyles => {
-  const { className, delay, maxWidth, theme } = props;
+  const { className, delay, gapSpace = 0, maxWidth, theme } = props;
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.styles.ts]: much nicer

**Comment by khmakoto** [comment]
@msft-github-bot wait 4 hours

**Pull request merged**